tests/drivers/spi/spi_loopback/testcase.yaml

```diff
@@ -13,10 +13,14 @@ tests:
   drivers.spi.loopback: {}
   drivers.spi.loopback.internal:
     filter: CONFIG_SPI_LOOPBACK_MODE_LOOP
+  drivers.spi.loopback.lpspi.dma:
+    filter: CONFIG_HAS_MCUX_EDMA
+    extra_configs:
+      - CONFIG_SPI_MCUX_LPSPI_DMA=y
   drivers.spi.loopback.rtio:
     extra_configs:
       - CONFIG_SPI_RTIO=y
-    platform_allow: tdk_robokit1
+    platform_allow: mimxrt1170_evk_cm7
```

Comment:
AFAIK if you add a platform_allow clause, now it won't run any boards other than RT1170, so I think this is wrong

Reply:
Sorry, this was not intended.